app/src/main/java/com/asfoundation/wallet/support/SupportNotificationWorker.kt

```diff
@@ -0,0 +1,96 @@
+package com.asfoundation.wallet.support
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.work.RxWorker
+import androidx.work.WorkerParameters
+import com.asf.wallet.R
+import com.asfoundation.wallet.di.WorkerKey
+import dagger.Binds
+import dagger.Module
+import dagger.multibindings.IntoMap
+import io.reactivex.Completable
+import io.reactivex.Single
+import javax.inject.Inject
+
+
+class SupportNotificationWorker @Inject constructor(private val context: Context,
```

Comment:
I would remove all ocurrences of SupportNotificationBroadcastReceiver.CONSTANT to use a static import.
sometimes it can improved readability